content/faq/how-to-publish.md

```diff
@@ -13,94 +13,91 @@ If you don't have LBRY yet, download it [here](https://lbry.io/get).
 
 6. Please make sure to check the option for mature audiences if your `Thumbnail` is categorized as NSFW (Not Safe For Work). Otherwise just click "Upload".
-![Select the Content to Upload](https://spee.ch/6/4-thumbnail44.jpeg)
 
 7. Under the `Price`, first, determine if you want to make your content free or set a price (in USD or LBC) per view.
-![Set Price](https://spee.ch/4/5-choose-a-price2.png)
+![Set Price](https://spee.ch/d/priceset.png)
 
-8. You have the option to select/create the channel you would like to publish the content under. If one isn't selected, it will be posted anonymously.
-![Select Channel or Anonymous](https://spee.ch/4/channel22.png)
+8. You have the option to select/create the channel you would like to publish the content under. If no channel is selected, it will be posted anonymously.
+![Select Channel or Anonymous](https://spee.ch/2/channel2.png)
 
 9. Type in the URL you want the content to be found under, along with a minimum of 0.0001 LBC deposit for the upload (current limit, may change in the future). If you are trying to outbid a user-friendly/common URL, the system will suggest a minimum bid to take over the content at that vanity URL. There may be a delay for this takeover. Check out the `#content` channel on our [Discord chat](https://chat.lbry.io) to see this information (search for your URL). For more details regarding the URL or bid, check out our [naming document](https://lbry.io/faq/naming).
-![Video URL and Deposit](https://spee.ch/e/8content-urlf.jpeg)
+![Video URL and Deposit](https://spee.ch/9/naming.png)
 
 10. Next, there are selections for `Maturity`, `Language`,  and `License`. The default values are set as follows: Maturity being unchecked, which means the content is safe for all audiences, Language is set to `English`, and the License is set to `None`.  If a change is needed, click the dropdown menus and select the appropriate choice. Please check the `Mature audience only` option if your content is NSFW.
 
-![publish](https://spee.ch/c/7-license-2-and-publish.jpeg)
+![publish](https://spee.ch/0/ma-la-li.png)
 
-11. Read and agree to the terms of service.
+You may want to read our terms of service.
 
-12. Click `Publish`.
-![Click Publish](https://spee.ch/2/publish.jpeg)
+Click `Publish`.
 
-13. The file will process in the background and will be added to the LBRY Blockchain. Larger files will take longer to upload depending on your network speed. Please leave LBRY running while your content is in the "pending confirmation" mode. Currently, this page will not automatically refresh. You can continue to use LBRY while the upload completes.
+
+The file will process in the background and will be added to the LBRY Blockchain. Larger files will take longer to upload depending on your network speed. Please leave LBRY running while your content is in the "pending confirmation" mode. Currently, this page will not automatically refresh. You can continue to use LBRY while the upload completes.
```

Comment:
It does refresh automatically now. We can mention the pending state and notification when confirmed.